shopelectro/tests/tests_commands.py

```diff
@@ -130,6 +139,23 @@ def test_update_or_create_tags(self):
         self.assertEqual(updated_groups_count + create_count, TagGroup.objects.count())
         self.assertEqual(updated_tags_count + create_count, Tag.objects.count())
 
+    # @todo #522:60m Fix doubled named tags creation.
+    #  Append some hash to tag_name if it's not unique.
+    #  Pair (group, name) for every tag should be unique.
+    #  See test below for expected logic details.
+    #  Resurrect test `shopelectro.tests.tests_views.CatalogTags#test_doubled_tag`
+    @unittest.expectedFailure
+    def test_create_double_named_tags(self):
+        """Two tags with the same name should have different slugs."""
+        # create two tags with the same name, but from different groups
+        tag_data = {
+            **get_tag_as_dict(group='First group', tag='Doubled tag'),
+            **get_tag_as_dict(group='Second group', tag='Doubled tag'),
+        }
+        update_tags.create_or_update(tag_data)
```

Comment:
Why do you associate this test with catalog updating command? I believe, that it will better to add unique constraint to `name` field of `Tag` model and move this test to `tests_models.py`

Reply:
@artemiy312 , y, that's what i've done implementing this subtask.
I'll propose the whole decision in this PR